code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -262,14 +264,15 @@ components:
           examples:
             response:
               value:
-                status: "401"
+                status: 401
                 code: UNAUTHENTICATED
                 message: Request not authenticated due to missing, invalid, or expired credentials
-    NotAuthenticatedByIpPermissionDenied:
+    PhoneNumberVerificationPermissionDenied:
```

Comment:
The more generic error descriptions are there, the better, so this change is good.